src/main.cpp

```diff
@@ -166,7 +166,7 @@ unique_ptr<CommandTable> commands(shared_ptr<Root> root) {
         {"remove_monitor", { monitors, &MonitorManager::removeMonitorCommand }},
         {"move_monitor",   monitors->byFirstArg(&Monitor::move_cmd, &Monitor::move_complete) } ,
         {"rename_monitor", monitors->byFirstArg(&Monitor::renameCommand, &Monitor::renameComplete) },
-        {"monitor_rect",   monitor_rect_command},
```

Comment:
Remove monitor_rect_command in .cpp file in line 367?

Reply:
Or is it still needed?

Reply:
Thanks! I simply forgot to remove it. This was probably the reason why the coverage decreased by 0.1%.

Reply:
Makes sense :D Looks good now, thanks!